FutureNorthants/northants-design-system
# Migrate stories to csf3

Resolves #541 

Migrate all stories from Component Story Format (CSF) v2 to v3. 

Follows the upgrade guide: https://storybook.js.org/docs/api/csf#upgrading-from-csf-2-to-csf-3

## Testing
- Checkout this branch `git fetch && git checkout 541-migrate-stories-to-csf3`
- Run `npm install`
- Then launch with `npm run dev` to manually test the stories still work as expected and show the same content as the live version
- Manually run unit tests with `npm run test`.
